pom.xml

```diff
@@ -224,6 +224,13 @@
         <type>pom</type>
         <scope>import</scope>
       </dependency>
+      <dependency>
+        <groupId>com.redhat.bom.rh-sso</groupId>
+        <artifactId>rh-sso-adapter-bom</artifactId>
+        <version>7.4.0.GA</version>
```

Comment:
Does the version match the one in EAP 7.2.2.GA? We are not on 7.3 yet...

Reply:
Also I would add a comment like "if changed version.keycloak value must be changed accordingly" to avoid the mess in the future.

Reply:
We are going to move to EAP 7.3 "soon". Show I merge it anyway or wait for the move to 7.3 EAP?